src/index.ts

```diff
@@ -51,6 +51,11 @@ export class CdktfProviderProject extends JsiiProject {
         distName: `${namespace}-cdktf-provider-${providerName}`,
         module: `${namespace}_cdktf_provider_${providerName}`,
       },
+      java: {
+        javaPackage: `${namespace}.cdktf.provider.${providerName}`,
```

Comment:
This is a bit awkward with the current namespace of `cdktf`

Reply:
Could you elaborate on what you mean?

Reply:
Since the current `namespace` is `cdktf`, we end up with `cdktf.cdktf.provider.providerName`